core/email.py

```diff
@@ -246,29 +250,31 @@ def sendEMail(*,
     return True
 
 
-def sendEMailToAdmins(subject: str, body: str, *args, **kwargs):
+def sendEMailToAdmins(subject: str, body: str, *args, **kwargs) -> bool:
     """
-        Sends an e-mail to the root users of the current app.
+    Sends an e-mail to the root users of the current app.
+
+    If conf["viur.email.admin_recipients"] is set, these recipients
+    will be used instead of the root users.
 
-        :param subject: Defines the subject of the message.
-        :param body: Defines the message body.
+    :param subject: Defines the subject of the message.
+    :param body: Defines the message body.
     """
     success = False
     try:
-        if "user" in dir(conf["viur.mainApp"]):
-            users = []
-            for userSkel in conf["viur.mainApp"].user.viewSkel().all().filter("access =", "root").fetch():
-                users.append(userSkel["name"])
-
-            if users:
-                ret = sendEMail(dests=users, stringTemplate=os.linesep.join((subject, body)), *args, **kwargs)
-                success = True
-                return ret
-            else:
-                logging.warning("There are no root-users.")
-
-    except Exception:
-        raise
+        users = []
+        if conf["viur.email.admin_recipients"] is not None:
+            users = normalize_to_list(conf["viur.email.admin_recipients"])
```

Comment:
This is already done by sendEMail
```suggestion
            users = conf["viur.email.admin_recipients"]
```

Reply:
If `users` is a `callable`, it will already be called at this point. If the result is empty, this method can fail-fast in https://github.com/viur-framework/viur-core/blob/72277865f4e5b0ab035b94acca2c76f38ef0a5db/core/email.py#L272

Also, if the callable `users` returns a string, it will be converted to a list in the `normalize_to_list` call inside `sendEMail`.

So I would keep it to be most flexible.